build/dependencies.props

```diff
@@ -1,10 +1,10 @@
 <Project>
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
-    <CoreFxVersion>4.3.0</CoreFxVersion>
+    <CoreFxVersion>4.4.0-*</CoreFxVersion>
```

Comment:
Ah I need to go update httpabstractions then. Yes we should be using 4.4.0. I don't think those packages were mirrored on our feeds when I started that pr and I meant to go back after the mirror builds finished.